phantomjs-env/src/test/scala/org/scalajs/jsenv/phantomjs/RetryingComJSSuite.scala

```diff
@@ -0,0 +1,25 @@
+package org.scalajs.jsenv.phantomjs
+
+import scala.concurrent.Future
+
+import org.scalajs.jsenv.nodejs.NodeJSEnv
+import org.scalajs.jsenv._
+import org.scalajs.jsenv.test._
+
+import org.junit.runner.RunWith
+
+@RunWith(classOf[JSEnvSuiteRunner])
+class RetryingComJSSuite extends JSEnvSuite(RetryingComJSSuite.Config)
+
+object RetryingComJSSuite {
+
+  private final val maxFails = 5
+
+  val Config = {
+    val jsEnv = new RetryingComJSEnv(new NodeJSEnv, maxFails)
+    JSEnvSuiteConfig(jsEnv)
+      .withTerminateVMJSCode("__ScalaJSEnv.exitFunction(0)")
```

Comment:
It seems like we should move this to the base test suite instead (under something like `withSupportsExit`).